DispatchGroup/ViewController.swift

```diff
@@ -59,7 +59,7 @@ class ViewController: UIViewController {
     
     override func viewDidLoad() {
         super.viewDidLoad()
-//        dispatchGroupCall()
+        //        dispatchGroupCall()
         operationQueue()
     }
     
```

Comment:
self.result3?.imageResults[0].image.src)! this is not safe. use guard let statement instead